packages/suite/package.json

```diff
@@ -29,6 +29,7 @@
         "file-saver": "^2.0.5",
         "framer-motion": "^3.2.1",
         "google-auth-library": "6.0.6",
+        "history": "^4.10.1",
```

Comment:
History in version 4 have at least 2 bugs.

First one is that if you click on one link more times, it is stored in browser history everytime. So if you click on Dashboard 5x, then if you hit 5x times back button, it is still dashboard. 
https://github.com/ReactTraining/history/pull/570#issuecomment-476511221

Another one is something with encoding strings into URL (for example for filtering), but I think we do not use that. If yes, I can try to find out what was the exact problem.

Reply:
first issue could be resolved by adding condition to `routerActions.goto` action, something like:
`if (currentRoute === requestedRoute) return;`